### PR TITLE
Release doc 050 fix

### PR DESCRIPTION
**Pull request by JamWils**
CX additions for the documentation.

### CODE REVIEW

File: versioned_docs/version-0.5.0/enterprise/upgrading.mdx [hunk 1]
```diff
@@ -98,12 +98,17 @@ Contact sales@weave.works for a valid entitlements secret. Then apply it to the
 kubectl apply -f entitlements.yaml
 ```
 
+In addition, you will add your git repository token (in this example, GitHub) as a secret. This is used to update the configurations and manifests in the management repository.
```
Comment (yiannistri) [inline versioned_docs/version-0.5.0/enterprise/upgrading.mdx:101]: s/will add/will need to add/ ?

Actually is this still required given that we can now do client side auth with GH?

**Review by yiannistri — COMMENTED** [review]
LGTM just wondering if we still need to add a git provider token.

**Review by yiannistri — APPROVED** [review]
Except for one minor suggestion this looks good 👍

File: versioned_docs/version-0.5.0/enterprise/upgrading.mdx (hunk 1 above)
Comment (pfcurtis) [reply versioned_docs/version-0.5.0/enterprise/upgrading.mdx:101]: As we're still seeing authorization loops, I would recommend this for now. The user experience isn't good with several auths required.

**Pull request merged**